prototype2/common/EFUArgs.h

```diff
@@ -24,10 +23,10 @@ struct ThreadCoreAffinitySetting {
 
 class EFUArgs {
 public:
-  enum class Status {EXIT, CONTINUE};
```

Comment:
Nooooo! and also why?

Reply:
This is the doing of clang-format (version 3.9 specifically).